test/Feature/Atomic.c

```diff
@@ -0,0 +1,69 @@
+// REQUIRES: geq-llvm-3.7
```

Comment:
Please remove this. This should hopefully not be needed.

Reply:
It is needed. It breaks on previous llvm versions.

Reply:
No the previous version was the one with test split in multiple files.